introduction.md

```diff
@@ -1 +1,8 @@
-Hello my name is Randall J Collins Jr and I am a programmer utilizing code school for learning. My favorite Code School path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?